pyproject.toml

```
@@ -70,7 +70,10 @@ matplotlib = ">=3.1,<3.4"
 attrs = ">=19.3,<21.3"
 jsonpickle = ">=1.3,<2.1"
 redis = "^3.4"
-numpy = ">=1.19.2,<1.20.0"
+
+numpy = ">=1.19.2,<1.22.2"
```

Just noting that this still pins the version to be < 1.22.2, which, as you found (in https://github.com/RasaHQ/rasa/issues/10816#issuecomment-1042447849) was fixed in 1.22.2. IMHO, a fix here would look something like:
```suggestion
numpy = ">=1.19.2,<1.23.0"
```

My ideal solution would be:
```suggestion
numpy = ">=1.22.2,<1.23.0"
```
to ensure _no one_ gets a CVE-riddled version of the library :).